tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
     "compilerOptions": {
         "target": "es5",
+        "outDir": "lib/",
         "module": "commonjs",
-        "noEmitHelpers": true,
+        "noEmitHelpers": false,
```

Comment:
If I remember correctly previously it was to prevent adding __extends twice. Either for core + duplex or duplex + puppetjs.
@alshakero Does the bundle with Palindrom and this branch of JSON-Patch has it added only once?